src/components/common/Dialogs/CreatePaymentDialog/CreatePaymentDialogForm.tsx

```diff
@@ -63,7 +63,7 @@ const MSG = defineMessages({
 });
 
 interface Props extends ActionDialogProps {
-  verifiedUsers: ColonyWatcher['user'][];
+  verifiedUsers: User[];
```

Comment:
Shouldn't this be `MemberUser` instead? You get those off `watchers` and `contributors` which don't fetch the entire User fragment